src/main/java/cloud/fogbow/ras/core/plugins/interoperability/azure/volume/AzureVolumePlugin.java

```diff
@@ -152,6 +177,17 @@ void updateInstanceAllocation(VolumeOrder volumeOrder) {
         }
     }
     
+    @VisibleForTesting
+    String defineResourceGroupName(Azure azure, String resourceName) {
```

Comment:
What do you think about this suggestion ?
```suggestion
    @VisibleForTesting
    String defineResourceGroupName(Azure azure, String resourceName) {
        try {
            resourceGroupName = AzureResourceGroupUtil.create(azure, this.defaultRegionName, resourceName);
        } catch (Exception e) {
            LOGGER.debug("", e)        
            LOGGER.info("Something the explain this normal and predicted situation"));
            return this.defaultResourceGroupName;
        }
    }
```